### PR TITLE
Token-gated Certificate

### PR DESCRIPTION
**Pull request by benoitdevos**
* Split authentication and download on collection item certificate.

logion-network/logion-internal#767

### CODE REVIEW

File: src/certificate/Certificate.tsx [hunk 1]
```diff
@@ -254,9 +259,33 @@ export default function Certificate() {
...
+                            <IntroductionText loc={ loc } tokenGated={ true } />
+                        </Col>
+                    </Row>
+                }
+                { !tokenGated &&
+                    <Row>
+                        <Col md={ 2 }/>
+                        <Col md={ 8 }>
+                            <IntroductionText loc={ loc } tokenGated={ false } />
+                        </Col>
```
Comment (gdethier) [inline src/certificate/Certificate.tsx:285]: This is a bit odd? What do you want to achieve exactly?

**Review by gdethier — APPROVED** [review]
Good job! I left one question.

File: src/certificate/Certificate.tsx (hunk 1 above)
Comment (benoitdevos) [reply src/certificate/Certificate.tsx:285]: Not sure to understand your question. If you refer to the attribute `tokenGated`, it's meant to adapt the introduction text when you are (or not) in a token-gated collection item certificate.

Comment (gdethier) [reply src/certificate/Certificate.tsx:285]: I was referring to the fact that you are using a `Col` as a left margin, which is not a good practice.

**Pull request merged**